### PR TITLE
Update Finetuning tasks

### PR DESCRIPTION
**Pull request by yawenzzzz**
This PR include updates on a bunch of our FT tasks (PR #122 and #130 have been merged in): 

rslearn datasets:

- [x] Kenya/Nandi crop type classification
- [x] WorldCereal cropland classification
- [x] Mangrove classification
- [x] Mangrove loss detection
- [x] LFMC
- [x] Landsat detection
- [x] Landsat classification

model configs: 
- [x] Kenya/Nandi crop type classification
- [x] WorldCereal cropland classification
- [x] Mangrove classification
- [x] Mangrove loss detection
- [x] LFMC
- [x] Landsat detection
- [x] Landsat classification

### CODE REVIEW

File: rslp/mangrove/create_windows_for_classification.py [hunk 1]
```diff
@@ -0,0 +1,204 @@
+"""Create windows for crop type mapping."""
```
Comment (favyen2) [inline rslp/mangrove/create_windows_for_classification.py:1]: Fix docstring

File: rslp/lfmc/create_windows.py [hunk 2]
```diff
@@ -0,0 +1,179 @@
+"""Create windows for cropland classification."""
```
Comment (favyen2) [inline rslp/lfmc/create_windows.py:1]: Fix docstring

File: rslp/scripts/beaker_launcher.py [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+            task_name=experiment_name,
+            beaker_image=image,
+            priority=Priority.high,
+            command=command,
+            datasets=[weka_mount],
+            resources=resources,
+            preemptible=True,
+            constraints=constraints,
+            env_vars=[
+                EnvVar(name="NASA_EARTHDATA_USERNAME", value="yawenz"),
```
Comment (favyen2) [inline rslp/scripts/beaker_launcher.py:98]: Recommend to reset your password for your EarthData account (since this PR is open source), and here please change it from hardcoded username/password to either read from Beaker secret or read from local environment variable.

File: rslp/scripts/beaker_launcher.py [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+    "64",
+    "--no-use-initial-job",
+    "--retry-max-attempts",
+    "8",
+    "--retry-backoff-seconds",
+    "60",
+    "--ignore-errors",
+]
+
+BEAKER_BUDGET = "ai2/d5"
```
Comment (favyen2) [inline rslp/scripts/beaker_launcher.py:33]: These constants can be imported from rslp.utils.beaker.

File: .gitignore [hunk 5]
```diff
@@ -2,3 +2,7 @@
 wandb/
 rslp/__pycache__/
 **__pycache__
+rslp/crop_type_mapping/csv/
```
Comment (favyen2) [inline .gitignore:5]: I don't think these would normally be created based on the README of the kenya_nandi/worldcereal/other projects, so it may be unnecessary to add to `.gitignore` here. Maybe instead you can add a `my_data` or similar folder to `.gitignore` and that can be used for files like these in the future.

**Review by favyen2 — APPROVED** [review]

File: data/helios/v2_landsat_vessels/finetune_detector.yaml [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+        split: "val"
+    test_config:
+      patch_size: 128
+      load_all_patches: true
+      groups: ["labels_utm"]
+      tags:
+        split: "val"
+trainer:
+  max_epochs: 200
+  callbacks:
```
Comment (favyen2) [inline data/helios/v2_landsat_vessels/finetune_detector.yaml:120]: Missing the freezing here. I think freezing the Helios encoder and then setting learning rate for encoder to 1/10 of the head learning rate helps on other tasks, but I'm not sure if it will help on this one, but that is the recipe I have been using for the other tasks.
